### PR TITLE
support passing arguments to the started executable

### PR DESCRIPTION
**Pull request by Bo0mer**
Updates #3.

### CODE REVIEW

File: internal/project/runner.go [hunk 1]
```diff
@@ -25,6 +28,7 @@ func (r *Runner) Run(ctx context.Context, path string) (*Process, error) {
 	cmd := exec.CommandContext(runCtx, path)
 	cmd.Stdout = output
 	cmd.Stderr = output
+	cmd.Args = append(cmd.Args, r.args...)
```
Comment (mokiat) [inline internal/project/runner.go]: What about passing the arguments as third argument to `exec.CommandContext` ?

**Review by mokiat — CHANGES_REQUESTED** [review]

File: internal/project/runner.go (hunk 1 above)
Comment (Bo0mer) [reply internal/project/runner.go]: Totally makes sense. I was not paying attention that this is possible.

File: main.go [hunk 2]
```diff
@@ -74,6 +81,7 @@ func main() {
 				ExcludeGlobs:    c.StringSlice("glob-exclude"),
 				RunDir:          c.String("run"),
 				CachedBuild:     c.String("cache"),
+				Args:            flag.ShlexStrings(c.Generic("args")),
```
Comment (Bo0mer) [inline main.go:84]: I don't find this very attractive, but was not able to come up with a shorter solution to hide the ugly conversion from interface{} to []string.

Comment (mokiat) [reply main.go:84]: Looks good to me.

**Review by mokiat — APPROVED** [review]

**Pull request merged**